assets/src/edit-story/panels/backgroundColor.js

```diff
@@ -24,17 +25,19 @@ function BackgroundColorPanel( { selectedElements, onSetProperties } ) {
 		evt.preventDefault();
 	};
 	return (
-		<Panel onSubmit={ handleSubmit }>
-			<Title>
+		<Panel>
+			<PanelTitle>
 				{ 'Background color' }
-			</Title>
-			<InputGroup
-				type="color"
-				label="Background color"
-				value={ state.backgroundColor }
-				isMultiple={ backgroundColor === '' }
-				onChange={ ( value ) => setState( { ...state, backgroundColor: value } ) }
-			/>
+			</PanelTitle>
+			<PanelContent onSubmit={ handleSubmit }>
+				<InputGroup
+					type="color"
+					label="Background color"
+					value={ state.backgroundColor }
+					isMultiple={ backgroundColor === '' }
+					onChange={ ( value ) => setState( { ...state, backgroundColor: value } ) }
+				/>
+			</PanelContent>
```

Comment:
Would it perhaps make sense to still have just single component for `Panel` and use it sth like this:
```
<Panel title={ ... } onSubmit={ ... }>
    <InputGroup .... />
</Panel>
```
So that a Panel would always have title and content and added in the same order in the `Panel` component without the using component having to worry about it. Or is there a case where only Panel or Title would be needed? Thoughts?

Reply:
Then Panel would suddenly receive a lot of different potential arguments, including the [`isPrimary` and `isResizable`](https://github.com/ampproject/amp-wp/pull/4138/files#diff-e076dce91853b2a885d114270731b71dR56) (used for title) and [`onSubmit` and `hasPadding`](https://github.com/ampproject/amp-wp/pull/4138/files#diff-8f60a4d61f7b188af469c59d9effcf7bR22) (used for content, the latter not added yet but needed to remove padding for layer panel). I think this keeps the components cleaner while not being too hard to use.

But it's a matter of taste. I prefer a higher number of small components rather than the opposite.

Come to think of it: We could add a `SimplePanel` wrapping these to take care of the basic and most common use case. That's actually a decent idea.

Reply:
Done in 6f2c4f434 (and a fix in d3839f820).